python/external_task_api_client/external_task_api_client_service.py

```diff
@@ -0,0 +1,71 @@
+import aiohttp
+import asyncio
+import json
+
+
+class ExternalTaskApiClientService:
+
+    def __init__(self, baseUrl):
+        self.__baseUrl = baseUrl
```

Comment:
Die URIs kann man gut hier ablegen, auch mit format string:

```suggestion
        self.__baseUrl = baseUrl
        self.__extend_lock_base_uri =  'task/{}/extend_lock'
```

Und dann kann man das so benutzen:

```python
uri = self.__extend_lock_base_uri.format(external_task_id)
```

Reply:
wenn man alle routen hier sammelt, macht es das einfach, die zu ändern/überblicken.